src/coreclr/pal/src/libunwind/src/CMakeLists.txt

```diff
@@ -153,6 +153,40 @@ SET(libunwind_elfxx_la_SOURCES
     elfxx.c
 )
 
+# The list of files that go into libunwind and libunwind-loongarch:
+SET(libunwind_la_SOURCES_loongarch_common
```

Comment:
Unless this results in build breakage, I would move changes from this file to https://github.com/dotnet/runtime/pull/62887 that way each PR would be self-contained.